src/main.rs

```diff
@@ -176,6 +176,19 @@ struct Args {
     /// If provided, allows to set a custom Max Execution time for the Host in ms.
     #[clap(long = "max-execution-time-ms", default_value = "600000", env = "WASMCLOUD_MAX_EXECUTION_TIME_MS", value_parser = parse_duration_millis)]
     max_execution_time: Duration,
+    /// The maximum amount of memory that a component can allocate
+    #[clap(long = "max-linear-memory", default_value_t = 10 * 1024 * 1024, env = "WASMCLOUD_MAX_LINEAR_MEMORY")]
+    max_linear_memory: u64,
+    /// The maximum size of a component binary that can be loaded
+    #[clap(long = "max-component-size", default_value_t = 50 * 1024 * 1024, env = "WASMCLOUD_MAX_COMPONENT_SIZE")]
```

Comment:
What do you think about dding the units for the param names (ex. `max-linear-memory-bytes`, etc)?

Reply:
Yeah I like that idea since it's not a human readable thing